protos/server.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto2";
+import "graph_structures.proto";
+package ujt_protos;
+
+service ReportingService {
+    // for generating graph
+    rpc GetNodes(NodeRequest) returns (NodeResponse) {}
+    // for updating statuses
+    rpc GetSLIs(SLIRequest) returns (SLIResponse) {}
+    // for user journeys
+    rpc GetClients(ClientRequest) returns (ClientResponse) {}
+
+    //rpc SetComment(CommentRequest) returns (CommentResponse) {}
+}
+
+message NodeRequest {
+    // add filtering fields later...
+    // not sure if it's necessary/useful to request a subset of nodes
+    // I imagine nodes should be requested once at startup (and perhaps force-refreshed)
+}
+
+message NodeResponse {
+    repeated Node nodes = 1;
+}
+
+message SLIRequest {
+    // same situation as NodeResponse, not sure when we need to request individual SLIs.
```

Comment:
nit: "same situation as NodeRequest" or "as GetNodes"?

Reply:
oops, should be NodeRequest